app/frontend/src/features/auth/login/UsernameForm.tsx

```diff
@@ -74,6 +74,7 @@ export default function UsernameForm() {
     <>
       <form onSubmit={onSubmit}>
         <TextField
+          id="username/email"
```

Comment:
```suggestion
          id="username-email"
```
I think the original is indeed valid html5 but having a slash isn't conventional I think.